amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpRpcFlow.scala

```diff
@@ -0,0 +1,30 @@
+package akka.stream.alpakka.amqp.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.amqp._
+import akka.stream.javadsl.{Flow, Source}
+import akka.util.ByteString
+
+import scala.concurrent.Future
+
+object AmqpRpcFlow {
+
+  /**
+    * Java API: Creates an [[AmqpRpcFlow]] with given settings and buffer size.
+    */
+  def create(settings: AmqpSinkSettings, bufferSize: Int) =
```

Comment:
Always use explicit return types on the public methods, all across the PR. So these should be `javadsl.Flow[...]`

Also, the scaladoc/javadoc here isn't very useful, try to explain what the flow it creates does rather than "it takes these parameters and returns a flow of this type" - that is already pretty much clear from the signature.

Something like "Create a RPC style flow (... link to rpc sample in rabbitmq docs...) allowing for bla bla communication over the message bus".